utils/doclint/check_public_api/index.js

```diff
@@ -176,8 +183,12 @@ function compareDocumentations(actual, expected) {
     const actualMethods = Array.from(actualClass.methods.keys()).sort();
     const expectedMethods = Array.from(expectedClass.methods.keys()).sort();
     const methodDiff = diff(actualMethods, expectedMethods);
-    for (const methodName of methodDiff.extra)
+    for (const methodName of methodDiff.extra) {
+      const missingMethodsForClass = expectedNonExistingMethods.get(className);
+      const methodNameIsExpectedMissing = missingMethodsForClass ? missingMethodsForClass.has(methodName) : false;
+      if (methodNameIsExpectedMissing) continue;
```

Comment:
How about something like:

```suggestion
      const missingMethodsForClass = expectedNonExistingMethods.get(className);
      if (!missingMethodsForClass) continue;
      if (missingMethodsForClass.has(methodName)) continue;
```

...or...

```
      const missingMethodsForClass = expectedNonExistingMethods.get(className);
      if (!missingMethodsForClass || missingMethodsForClass.has(methodName)) continue;
```

I think the former is most readable, though. WDYT?